docs/common-practices/tutorial-tilt-gestures.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+title: 'Creating Tilt Gestures'
+parent: Common practices
+nav_order: 27
+---
+
+# Creating Tilt Gestures
+
+The Puara **Tilt** processor is a powerful tool for interpreting real-world motion as a rotational gesture. It intelligently combines data from multiple sensors to calculate a device's forward-and-backward tilt (its "pitch" angle).
+
+This tutorial will guide you through a complete example. We will use a smartphone as a live sensor to provide data to the Tilt processor and visualize its output. The principles apply to any hardware that provides accelerometer and gyroscope.
+
+### Part 1: Setting Up the Live Sensor Data
+
+To see the Tilt processor in action, we first need a source of motion data. We'll use the [Sensors2OSC](https://sensors2.org) app to stream data from a phone over Wi-Fi.
```

Comment:
Same as other docs that use it : please add an Android disclaimer (and maybe an alternative for iPhone)